curriculum/challenges/english/09-information-security/information-security-projects/stock-price-checker.md

```diff
@@ -62,7 +62,7 @@ async (getUserInput) => {
 };
 ```
 
-You can send a `GET` request to `/api/stock-prices` with form data containing a NASDAQ stock ticker. Returned will be an object with the property `stockData`.
+You can send a `GET` request to `/api/stock-prices` with the query parameter `stock` representing a NASDAQ stock ticker. Returned will be an object with the property `stockData`.
```

Comment:
```suggestion
You can send a `GET` request to `/api/stock-prices`, passing a NASDAQ stock symbol to a `stock` query parameter. Returned will be an object with the property `stockData`.
```

Reply:
Regarding the first sentence, I feel indifferent, so I think it's fine.
I think the second sentence is much better now.

While we're suggestion other changes, may I suggest linking `NASDAQ stock symbol` to the Wikipedia page for [Ticker symbol](https://en.wikipedia.org/wiki/Ticker_symbol)?

English: https://en.wikipedia.org/wiki/Ticker_symbol
Spanish: https://es.wikipedia.org/wiki/S%C3%ADmbolo_burs%C3%A1til
Chinese: https://zh.wikipedia.org/wiki/%E8%82%A1%E7%A5%A8%E4%BB%A3%E8%99%9F

I say this as someone that didn't know what a NASDAQ ticker/stock symbol was until looking.

Reply:
Actually sorry, I do have a question/comment.

With "NASDAQ stock ticker" changing to "NASDAQ stock symbol", I support the change as it's the term used on Wikipedia, but I think the language of this file should be standardized.

There is one other place that uses the term "stock ticker", shall I change this to "stock symbol" as well?

> The `stockData` property includes the `stock` ticker as a string, the price as a number, and `likes` as a number.

would become 

> The `stockData` property includes the `stock` symbol as a string, the price as a number, and `likes` as a number.